.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lib"]
 	path = lib
-	url = https://github.com/libusual/libusual.git
+	url = https://github.com/enocom/libusual.git
```

Comment:
Leaving this in place so the PR will build. Assuming the libusual PR is acceptable, I'll swap this back to its proper path next.